Epona/include/Epona/JacobiEigenvalue.hpp

```diff
@@ -23,37 +23,37 @@ class JacobiEigenvalue
      * @param[in] maxIterations
      */
     explicit JacobiEigenvalue(
-        glm::dmat3 const& symmetricMatrix,
-        double coverageThreshold = 1.0e-4,
+        glm::mat3 const& symmetricMatrix,
+        float coverageThreshold = 1.0e-4f,
```

Comment:
Should this relate to `fp::g_floatingPointThreshold` or is it completely separate independent value?

Reply:
As I vaguely remember that probably should.